### PR TITLE
[QA-16072] Fix integrazione attributi certificati discreti

### PR DESCRIPTION
**Pull request by manu71**

### CODE REVIEW

File: interop-qa-tests/src/test/java/it/pagopa/pn/interop/cucumber/steps/attribute/AttributeListingSteps.java [hunk 1]
```diff
@@ -122,4 +114,34 @@ private void listAttributes(AttributeListRequest attributeListRequest) {
...
+                    )
+            );
+
+            Assertions.assertTrue(httpCallExecutor.getResponseStatus().is2xxSuccessful(), "Expected 2xx successful status code for attribute listing");
+
+            Attributes response = (Attributes) httpCallExecutor.getResponse();
+            allAttributes.addAll(response.getResults());
+            totalCount = response.getPagination().getTotalCount();
+            offset += limit;
+        } while (allAttributes.size() < totalCount);
```
[the remark this answers is not in the document]
Comment (manu71) [reply interop-qa-tests/src/test/java/it/pagopa/pn/interop/cucumber/steps/attribute/AttributeListingSteps.java:140]: this is an edge case we don’t want to handle here. We’re assuming the API returns consistent pagination and non-null fields.

**Review by giuseppe-veldorale — APPROVED** [review]
